ibm_cloud_sdk_core/base_service.py

```diff
@@ -82,9 +84,14 @@ def __init__(self,
                  service_url: str = None,
                  authenticator: Authenticator = None,
                  disable_ssl_verification: bool = False,
-                 enable_gzip_compression: bool = False) -> None:
+                 enable_gzip_compression: bool = False,
+                 max_retries: int = 10,
```

Comment:
I'm leaning toward the notion that the max_retries and retry_interval parameters should not be included in the ctor.  Instead, let's introduce two methods:
1. `enable_retries(max_retries, retry_interval)`
2. `disable_retries()`
This is simlar to the way in which the Go core works.